.travis.yml

```diff
@@ -11,8 +11,9 @@ android:
     - extra-google-m2repository
     - addon-google_apis-google-16
 before_script:
-  - cd actionlist && npm install
+  - cd actionlist && rm -rf node_modules && npm install
```

Comment:
In that case, there should be a comment explaining why file system operations are occurring, hopefully with a link to an outstanding issue in the offending dependency. In this case: https://github.com/facebook/jest/issues/2070